mentors/Makefile

```diff
@@ -1,7 +1,7 @@
 DOCKER_IMAGE?=mentor-processing-pipeline
 DOCKER_IMAGE_ID=$(shell docker images -q $(DOCKER_IMAGE))
 DOCKER_CONTAINER=mentor-processing-pipeline
-MENTOR?=julianne
+MENTOR?=juliannenordhagen
```

Comment:
curious why full name here? It actually does make better sense as ID, but I think everywhere downstream we use just 'juliannne'

Reply:
Well I was thinking that since we are migrating from drive to s3, I could use this opportunity to rename julianne's folders from `julianne` to `juliannenordhagen`. That would help with the ID, and also the questions-paraphrases and prompts-utterances files currently have a mentor column that uses the full name.

Reply:
Would this break the classifier?

Reply:
Yeah, we will have to update it a bunch of downstream places. For now, I would just use `julianne`

What should I do to test this locally?